functions.php

```diff
@@ -53,4 +54,58 @@ function displayGames(array $games): string
     return $result;
 }
 
+/**
+ * validate form inputs
+ *
+ * @param string $name
+ * @param string $year
+ * @param string $developer
+ * @param string $imdb_rating
+ * @param string $image_url
+ * @return string true if all form inputs are valid, false if not
+ */
+function validateFormInput(string $name, string $year, string $developer, string $imdb_rating, string $image_url): string
+{
+    $validName = filter_var($name, FILTER_SANITIZE_STRING);
+    $validName = preg_match('/^[a-zA-Z0-9: ]{1,255}$/', $name);
```

Comment:
this regex lets an entry made of loads of space bars through. might want an extra check to make sure its not that after ypou have done this